### ISSUE TITLE
Blank test suite names and other missing text

### ISSUE THREAD

**Issue by chrismbarr** [razvanz/jasmine2-reporter#13]
I am getting some strange output using this reporter...  note the missing test suite names and the `undefined` that appears at the top and in the summary "title box" at the bottom

![image](https://cloud.githubusercontent.com/assets/463685/21570580/51657aaa-ce95-11e6-97bc-378964c89afe.png)

The tests run, and the pass/fail exactly as expected.  It's just that the reporter is missing some text and I don't understand why.

---

Here's everything about how I'm setting things up:

* Windows 10
* Node 6.9.1
* NPM 3.10.9

### `package.json` (relevant portion)
```json
"devDependencies": {
  "colors": "^1.1.2",
  "gulp": "^3.9.1",
  "gulp-jasmine": "^2.4.2",
  "jasmine2-reporter": "^0.2.0",
}
```

### `gulpfile.js` task
```js
const gulp         = require("gulp");
const jasmine      = require("gulp-jasmine");
const testReporter = require("jasmine2-reporter");

gulp.task("tests-app", () => {
    return gulp.src("./tests/app/**/*.tests.js")
        .pipe(jasmine({
            reporter: new testReporter.Jasmine2Reporter({
                stacktrace: false,
                failuresSummary: false
            })
        }));
});
```

### Test sample
totally normal...
```js
describe("All HTML templates", () => {
  it("should not contain more than one ng-controller", () => {
     if (!someCondition) {
        fail("message here");
     }
  });
});
```

---

## My troubleshooting so far & how I run the tasks

The weird part is that I have no issues with doing the exact same thing (even the same tests) in other projects at work.  I've tried deleting my `node_modules` folder and running `npm install` again, but I still get the same result.

I get the same results when I run the gulp tasks from the Windows command prompt and from the Visual Studio 2015 Task Runner

---

## Ideas

Looking into the source code, each place where `undefined` appears it seems to use `"string".bold.inverse` - and then I noticed that this package uses `"colors": "^1.0.1"` while my project uses a newer version (`"^1.1.2"`) - perhaps there is a version conflict here?

I've also tried this with the option `inColors: false` set - I get the same results, but just in plain white text.

**Comment by chrismbarr** [comment]
Further investigation... I dove into the source code locally and tried a few things.

`' Summary '.bold.inverse`   => `undefined`
`' Summary '.bold` => _empty string_
`' Summary '.inverse` => renders plain text (no bg color, that seems to now work in Windows anyway)

I changed the syntax to `colors.bold.inverse(' Summary ')` and now it works just fine!


Similarly I changed this functon
```js
getSuiteName: function (suite, color) {
  return this.options.namesInColors ?
      suite.description.bold[color] : suite.description.bold;
},
```

to this:
```js
getSuiteName: function (suite, color) {
  return this.options.namesInColors ?
      colors.bold[color](suite.description.bold) : colors.bold(suite.description);
},
```
and now the reporter displays as expected!

**Comment by razvanz** [comment]
Fixed with release `0.3.0`.
